pyaedt/emit_core/Couplings.py

```diff
@@ -0,0 +1,74 @@
+"""
```

Comment:
I would suggest to not create another folder. I think this file could stay in modules or modeler folders.

Reply:
@maxcapodi78 @myoung301, we thought we would group all EMIT only files under this emit_core foulder, this Coupling.py is the first one, there will be files for result nodes, and more